packages/espressocash_app/lib/features/outgoing_dln_payments/screens/flow_screen.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/material.dart';
+
+import '../../../../routes.gr.dart';
+import '../../../../ui/app_bar.dart';
+import '../../../../ui/theme.dart';
+import '../../../core/amount.dart';
+import '../../../di.dart';
+import '../../../l10n/l10n.dart';
+import '../../../ui/loader.dart';
+import '../../blockchain/models/blockchain.dart';
+import '../models/payment_quote.dart';
+import '../services/dln_order_service.dart';
+import 'confirmation_screen.dart';
+import 'details_screen.dart';
+
+@RoutePage()
+class OutgoingDlnPaymentFlowScreen extends StatefulWidget {
```

Comment:
`Flow` assumes that it has several screens underneath with its own routing logic. Here, it's more like a normal screen.